examples/readme.md

```diff
@@ -32,6 +32,12 @@ Then compile with `make BOARD=[your_board] all`, for example
 $ make BOARD=feather_nrf52840_express all
 ```
 
+If a board provides 2 USB ports, port 0 is selected by default. Add `PORT=1' to compile for PORT1
```

Comment:
This is a nice idea to overwrite the default port. but It is not correct that PORT 0 is selected by default, it depends on the BSP implementation of `board.mk` . It will normally pick the preferred one, mostly is OTG HS (port1) on available board.  Therefore I think it is better to implement it in `board.mk` as follows

```
ifneq ($(PORT),)
  PORT = 0 # or 1 depending on the individual board implementation
endif

CFLAGS += -DBOARD_DEVICE_RHPORT_NUM=$(PORT)

```

Reply:
Where is
```
PORT ?= 0
```
different from
```
ifneq ($(PORT),)
  PORT = 0 # or 1 depending on the individual board implementation
endif
```
I have adapted the readme however

Reply:
true, it is the same, only need to update the readme is good enough .